Rust/tests/samplesummarytest.rs

```diff
@@ -25,8 +24,6 @@ fn core(
     seed: u64,
     distance: fn(&[f32], &[f32]) -> f64,
 ) -> bool {
-    let mut rng = ChaCha20Rng::seed_from_u64(seed);
-    let data_size = 200000;
```

Comment:
This changes the data size to `1000000` for the parameterized test, but we can update it there if that's not desired.

Reply:
Should be fine with the change!